test-data/run.test

```diff
@@ -301,6 +301,30 @@ for k in range(12):
     assert next(gen) == k % 4
 [out]
 
+[case testAsync]
+import asyncio
+import asyncio.runners
```

Comment:
As a stopgap workaround for the module reference bug, add some of the submodules to the map of import hacks: https://github.com/mypyc/mypyc/blob/master/mypyc/genops.py#L1487-L1492

Reply:
Done